pkg/snowflake/password_policy.go

```diff
@@ -0,0 +1,114 @@
+package snowflake
+
+import (
+	"database/sql"
+	"reflect"
+)
+
+type PasswordPolicy struct {
+	SchemaObjectIdentifier
+
+	MinLength           int `pos:"parameter" db:"PASSWORD_MIN_LENGTH"`
+	MinLengthOk         bool
+	MaxLength           int `pos:"parameter" db:"PASSWORD_MAX_LENGTH"`
+	MaxLengthOk         bool
+	MinUpperCaseChars   int `pos:"parameter" db:"PASSWORD_MIN_UPPER_CASE_CHARS"`
+	MinUpperCaseCharsOk bool
+	MinLowerCaseChars   int `pos:"parameter" db:"PASSWORD_MIN_LOWER_CASE_CHARS"`
+	MinLowerCaseCharsOk bool
+	MinNumericChars     int `pos:"parameter" db:"PASSWORD_MIN_NUMERIC_CHARS"`
+	MinNumericCharsOk   bool
+	MinSpecialChars     int `pos:"parameter" db:"PASSWORD_MIN_SPECIAL_CHARS"`
+	MinSpecialCharsOk   bool
+	MaxAgeDays          int `pos:"parameter" db:"PASSWORD_MAX_AGE_DAYS"`
+	MaxAgeDaysOk        bool
+	MaxRetries          int `pos:"parameter" db:"PASSWORD_MAX_RETRIES"`
+	MaxRetriesOk        bool
+	LockoutTimeMins     int `pos:"parameter" db:"PASSWORD_LOCKOUT_TIME_MINS"`
+	LockoutTimeMinsOk   bool
+
+	Comment   string `pos:"parameter" db:"COMMENT"`
+	CommentOk bool
+}
+
+type PasswordPolicyManager struct {
+	BaseManager
+}
+
+func NewPasswordPolicyManager() (*PasswordPolicyManager, error) {
+	builder, err := newBuilder(
+		"PASSWORD POLICY",
+		"PASSWORD POLICIES",
+		reflect.TypeOf(PasswordPolicyCreateInput{}),
+		reflect.TypeOf(PasswordPolicyUpdateInput{}),
+		reflect.TypeOf(PasswordPolicyUpdateInput{}),
+		reflect.TypeOf(PasswordPolicyDeleteInput{}),
```

Comment:
i really don't like this way of registering functions with the password policy manager. i am sure there is a better way to do this.